src/window.ui

```diff
@@ -349,7 +349,7 @@
                 <child>
                   <object class="GtkToggleButton" id="shuffle">
                     <property name="visible">True</property>
-                    <property name="can-focus">False</property>
+                    <property name="can-focus">True</property>
```

Comment:
Is this intentional? Probably since can-focus makes sense here, but I'd rather have this in another PR (there are probably other focus/keyboard navigation issues in the main window too)

Reply:
It was the only one I noticed in the main window so thought I would throw it in with these. I can remove it if you want though.

Reply:
Okay, why not :)